client/src/tabular.ts

```diff
@@ -0,0 +1,110 @@
+import {User} from "./app.js";
+import {Role,referType,columnName} from "./models/model.js";
+
+let apiData: Array<object> = [];
+let usersData: Array<Array<string>> = [];
+let columnList: Array<string> = [];
+
+// this api function is used to fetch the user data from the server
+function api<T>(url: string): Promise<T> {
+    return fetch(url)
+      .then(response => {
+        return response.json()
+      })
+  
+  }
+
+// this load_button element execute function when click on load_button and fetch the data from server
+document.getElementById("load_button")?.addEventListener("click",async()=>{
+    let data: Array<object> = await api("http://localhost:9010/user");
```

Comment:
use dotenv package to store configurable parameters in environment file.
use these at other relevant places also, like DB config

Reply:
use the .env file to store config parameters